tests/base.py

```diff
@@ -343,6 +344,15 @@ def align_date_type(self, record, key, value):
             iso_date = dt.strftime(raw_date,  "%Y-%m-%dT%H:%M:%S.%fZ")
             record[key] = iso_date
 
+    def align_number_type(self, record, key, value):
+        """float values must conform to json number formatting so we convert to Decimal"""
+        if isinstance(value, float) and key in ['latitude', 'longitude']:
+            try:
+                record[key] = str(value)
+            except Exception as ex:
+                print("Failed to convert {} from type float to string".format(value))
```

Comment:
What is the purpose of this print? Wouldn't the exception have all the info required?

Reply:
Ah I don't think that is really needed now. It was there because for about 5 minutes because I was converting float to Decimal and wanted to log in case it failed. Don't think there will be an issue going from float to string.

Reply:
Removed.